cloud/firestore/v1/document_event_data.go

```diff
@@ -35,32 +35,32 @@ type OldValue struct {
 
 // A message that can hold any of the supported value types.
 type OldValueField struct {
-	ArrayValue     *ArrayValue    `json:"arrayValue,omitempty"`    // An array value.; ; Cannot directly contain another array value, though can contain an; map which contains another array.
-	BooleanValue   *bool          `json:"booleanValue,omitempty"`  // A boolean value.
-	BytesValue     *string        `json:"bytesValue,omitempty"`    // A bytes value.; ; Must not exceed 1 MiB - 89 bytes.; Only the first 1,500 bytes are considered by queries.
-	DoubleValue    *float64       `json:"doubleValue,omitempty"`   // A double value.
-	GeoPointValue  *GeoPointValue `json:"geoPointValue,omitempty"` // A geo point value representing a point on the surface of Earth.
-	IntegerValue   *int64         `json:"integerValue,omitempty"`  // An integer value.
-	MapValue       *MapValue      `json:"mapValue,omitempty"`      // A map value.
-	NullValue      *NullValue     `json:"nullValue"`               // A null value.
-	ReferenceValue *string        `json:"referenceValue,omitempty"`// A reference to a document. For example:; `projects/{project_id}/databases/{database_id}/documents/{document_path}`.
-	StringValue    *string        `json:"stringValue,omitempty"`   // A string value.; ; The string, represented as UTF-8, must not exceed 1 MiB - 89 bytes.; Only the first 1,500 bytes of the UTF-8 representation are considered by; queries.
-	TimestampValue *string        `json:"timestampValue,omitempty"`// A timestamp value.; ; Precise only to microseconds. When stored, any additional precision is; rounded down.
+	ArrayValue     *ArrayValue     `json:"arrayValue,omitempty"`    // An array value.; ; Cannot directly contain another array value, though can contain an; map which contains another array.
+	BooleanValue   *bool           `json:"booleanValue,omitempty"`  // A boolean value.
+	BytesValue     *string         `json:"bytesValue,omitempty"`    // A bytes value.; ; Must not exceed 1 MiB - 89 bytes.; Only the first 1,500 bytes are considered by queries.
+	DoubleValue    *float64        `json:"doubleValue,omitempty"`   // A double value.
+	GeoPointValue  *GeoPointValue  `json:"geoPointValue,omitempty"` // A geo point value representing a point on the surface of Earth.
+	IntegerValue   *int64          `json:"integerValue,omitempty"`  // An integer value.
+	MapValue       *MapValue       `json:"mapValue,omitempty"`      // A map value.
+	NullValue      *NullValueUnion `json:"nullValue"`               // A null value.
```

Comment:
These type name changes for exported types are a compatibility break.

Reply:
Yes. Is there a process or note of stableness you'd like to see in the repo?

Reply:
Ideally we use similar practices to other GCP libraries so developers can become familiar with those indicators.

Reply:
Also, per https://www.conventionalcommits.org/en/v1.0.0/, it would be good to have the PR title reflect the expectation of a breaking change like `feat!: ...`